docs/topics/RPC.md

```diff
@@ -259,6 +259,9 @@ Used to authenticate an existing client with your app.
 
 Used to get a list of guilds the client is in.
 
+> info
+> The response only contains the `id`, `name` and `icon_url` properties for each [Guild](#DOCS_RESOURCES_GUILD/guild-object)
```

Comment:
Is this not implied by the [Get Guild Response Structure](https://github.com/discord/discord-api-docs/pull/5050/files#diff-51a620c0b55efe72f5232e547f415fc43247ede707ff3c13f8db4112075c42a7R309)?

If these are talking about the same thing, it may make sense to link to that object rather than `#DOCS_RESOURCES_GUILD/guild-object` in the relevant places instead of adding a higher-level callout

Reply:
this is rpc so it's separate

(also I think that link is wrong)

Reply:
uhh it was just supposed to link to the code for https://discord.com/developers/docs/topics/rpc#getguild-get-guild-response-structure, which is still RPC

Reply:
oh, yeah it should prob just link to that section then

Reply:
I think that we should have a table for the response, just like the API.